CefSharp/IMenuModel.cs

```diff
@@ -4,8 +4,13 @@
 
 namespace CefSharp
 {
-	public interface IMenuModel
-	{
-		bool Clear();
-	}
+    public interface IMenuModel
+    {
+        int Count { get; }
```

Comment:
Will do my best. Trapped in other stuff right now.

Regards,
Zoltan

From: Alex Maitland [mailto:notifications@github.com]
Sent: Wednesday, August 26, 2015 8:30 AM
To: cefsharp/CefSharp <CefSharp@noreply.github.com>
Cc: Zoltan Kovacs <zkovacs@smallfootprint.com>
Subject: Re: [CefSharp] Exposing more methods in IMenuModel to facilitate context menu custom… (#1219)


In CefSharp/IMenuModel.cs<https://github.com/cefsharp/CefSharp/pull/1219#discussion_r37947408>:

> @@ -4,8 +4,13 @@

>

>  namespace CefSharp

>  {

> -     public interface IMenuModel

> -     {

> -            bool Clear();

> -     }

> +    public interface IMenuModel

> +    {

> +        int Count { get; }

Do you mind adding xml documentation to these? Taking to improve documentation as a whole [:+1:]

—
Reply to this email directly or view it on GitHub<https://github.com/cefsharp/CefSharp/pull/1219/files#r37947408>.